apps/andark/app.js

```diff
@@ -62,41 +61,42 @@ let drawHands = function(d) {
   g.fillCircle(c.x,c.y,4);
 };
 
-let drawText = function(d) {
+const drawText = function(d) {
   g.setFont("Vector",10);
   g.setBgColor(0,0,0);
   g.setColor(1,1,1);
-  let dateStr = require("locale").date(d);
+  const dateStr = require("locale").date(d);
   g.drawString(dateStr, c.x, c.y+20, true);
-  let batStr = Math.round(E.getBattery()/5)*5+"%";
+  const batStr = Math.round(E.getBattery()/5)*5+"%";
   if (Bangle.isCharging()) {
     g.setBgColor(1,0,0);
   }
   g.drawString(batStr, c.x, c.y+40, true);
 };
 
-let drawNumbers = function() {
+const drawNumbers = function() {
   //draws the numbers on the screen
   g.setFont("Vector",20);
   g.setColor(1,1,1);
   g.setBgColor(0,0,0);
   for(let i = 0;i<12;i++){
-     g.drawString(zahlpos[i][0],zahlpos[i][1],zahlpos[i][2],true);
+    g.drawString(zahlpos[i][0],zahlpos[i][1],zahlpos[i][2],true);
   }
 };
 
-let drawTimeout;
+let drawTimeout = undefined;
```

Comment:
Out of curiosity, what's the reason for being explicit here. Does the behavior change?

Reply:
I just read in description of `let` that unlike `var` reading of uninitialised `let` may lead to an error instead of just an `undefined` value. I don't know if espruino interpreter does this, but I decided to be on the safe compatible side

Reply:
You're thinking of hoisting and the [temporal dead zone](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let#temporal_dead_zone_tdz) - this is fine:
```javascript
let a;

console.log(a);
```

This is not:
```javascript
console.log(a);

let a;
```

The code you have works either way though :)

Reply:
Oh, yes, I see that I misread the specification and did useless thing. I deleted this piece and pushed the change.

Reply:
No problem - ok I think we can merge if you're happy @thyttan ?

Reply:
I'll go ahead then!